docs/src/modeling/forms/fields/README.md

```diff
@@ -186,20 +186,31 @@ Es gibt lediglich einen Unterschied bei der Auswahl and Anzeigemöglichkeiten.
 
 ![Formular Editor](~@source/modeling/forms/fields/files-context.png)
 
-::: tip
-Je nachdem ob es sich um ein Start- oder Benutzerformular handelt, muss der Zugriff zusätzlich konfiguriert werden.
-Dies kann in der jeweiligen Dokumentation nachgelesen werden
+#### Konfiguration
+
+Das Datei-Upload-Feld benötigt eine zusätzliche Konfiguration im Usertask des Prozesses, damit Dateien gespeichert und gelesen werden können.
+
+In den Inputparameter `app_file_paths_readonly` und `app_file_paths` müssen im Usertask der/die Ordner angegeben werden, auf die der User zugreifen darf.
```

Comment:
Das ist vielleicht etwas verwirrend, da in der CoCreation die so nicht heißen:

![image](https://github.com/it-at-m/digiwf-core/assets/10849776/88c54aaf-84b7-4af5-a789-bf1f8816231e)